doc/ivshmem_kvmfr.rst

```diff
@@ -120,12 +120,17 @@ change its ownership manually, i.e.:
 
    sudo chown user:kvm /dev/kvmfr0
 
-As an example, you can create a new file in ``/etc/udev/rules.d/99-kvmfr.rules``
+(replace ``user`` with your username)
+
+As an example, you can create a new file in ``/etc/udev/rules.d/70-kvmfr.rules``
 with the following contents::
 
-   SUBSYSTEM=="kvmfr", OWNER="user", GROUP="kvm", MODE="0660"
+   SUBSYSTEM=="kvmfr", GROUP="kvm", MODE="0660", TAG+="uaccess"
```

Comment:
This works fine on 257, I am using it right now. (As should be expected, since this has been in systemd for 13 years)